src/webpack/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
```

Comment:
Tapestry handles `/public/favicon.ico` by default so we good default to the Tapestry Logo? Might be a good look for dev mode.